### ISSUE TITLE
adjustment of training parameters

### ISSUE THREAD

**Issue by ljc19940403** [backseason/PoolNet#47]
Thank you for your excellent work,I recently trained the model on other datasets. In addition to the default parameter settings, do you have any suggestions for training tuning?
Looking forward to your reply.

**Comment by backseason** [comment]
Thanks for you attention.

The most important hyper-parameters I usually tune are learning rate, weight decay, and training epochs. 

I only drop learning rate once time and an extra drop usually shows no help.

As Adam is not very sensitive to the initial learning rate, I usually choose the biggest working one (the loss does not explode).

More training epochs sometime helps when the initial learning is slightly small.

Hope the above helps.

**Comment by ljc19940403** [comment]
> 感谢您的关注。
> 
> 我通常会调整的最重要的超参数是学习速度，体重减轻和训练时期。
> 
> 我只降低一次学习率，而额外的下降通常无济于事。
> 
> 由于亚当对最初的学习速度不是很敏感，因此我通常选择最大的学习方式（损失不会爆炸）。
> 
> 当最初的学习量很小时，有时会有更多的训练时间会有所帮助。
> 
> 希望以上内容对您有所帮助。



> Thanks for you attention.
> 
> The most important hyper-parameters I usually tune are learning rate, weight decay, and training epochs.
> 
> I only drop learning rate once time and an extra drop usually shows no help.
> 
> As Adam is not very sensitive to the initial learning rate, I usually choose the biggest working one (the loss does not explode).
> 
> More training epochs sometime helps when the initial learning is slightly small.
> 
> Hope the above helps.

Thank you for your reply, which is helpful to my work
